### PR TITLE
Add CI + Travis to build pipeline

### PR DESCRIPTION
**Pull request by lzchen**
PR includes running tests, coverage and linting. Part of [#6]

TODO: mypy for typing, docs

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+
+# List of qualified module names which can have objects that can redefine
+# builtins.
+redefining-builtins-modules=six.moves,past.builtins,future.builtins,builtins,io
+
+
+[FORMAT]
+
+# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
+expected-line-ending-format=LF
```
Comment (hectorhdzg) [inline .pylintrc]: CRLF

File: scripts/pylint.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019, OpenCensus Authors
```
Comment (hectorhdzg) [inline scripts/pylint.sh]: OpenCensus?

File: azure_monitor/tests/__init__.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
```
Comment (hectorhdzg) [inline azure_monitor/tests/__init__.py:2]: I thought the code in this repo was using Apache license

File: .flake8 [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  .git
+  .hg
+  .svn
+  .tox
+  CVS
+  .venv*/
+  venv*/
+  target
+  __pycache__
+  ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/gen/
```
Comment (hectorhdzg) [inline .flake8]: Not needed here

File: azure_monitor/tests/__init__.py (hunk 3 above)
Comment (lzchen) [reply azure_monitor/tests/__init__.py:2]: Any repo under a `Microsoft` org can use the MIT license as long as it was created by a Microsoft employee. This was discussed with a CELA rep.

File: .pylintrc (hunk 1 above)
Comment (lzchen) [reply .pylintrc]: LF for linux envioronment since our venv runs in ubuntu.

File: azure_monitor/tests/__init__.py (hunk 3 above)
Comment (hectorhdzg) [reply azure_monitor/tests/__init__.py:2]: I noticed we have multiple license headers in different files, do we want to use a single one or do we need to have all of them?

Comment (lzchen) [reply azure_monitor/tests/__init__.py:2]: We should be using MIT license consistently. Any other license used would be a bug.

**Review by hectorhdzg — APPROVED** [review]

**Pull request merged**